tools/DevicesApiTester/Commands/Pwm/DimLed.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Pwm;
+using System.Threading.Tasks;
+using CommandLine;
+using DeviceApiTester.Infrastructure;
+
+namespace DeviceApiTester.Commands.Pwm
+{
+    [Verb("pwm-dim-led", HelpText = "Dims an LED connected to a specified PWM pin for 3 seconds")]

```

Comment:
>pwm-dim-led [](start = 11, length = 11)

I think this should probably be generic.  It works with an LED, but it could work with any PWM device.  

Consider something alternate

Reply:
sample has been changed to allow generic output